models/models.go

```diff
@@ -352,3 +383,11 @@ func DumpDatabase(filePath string, dbType string) error {
 	}
 	return x.DumpTablesToFile(tbs, filePath)
 }
+
+func schenaQuery(sql string) string {
```

Comment:
Is `schena` a typo? It is used elsewhere so it might not be, but I feel as though this should have an `m` not an `n`.

Reply:
this is a typo.

Reply:
See: https://github.com/go-gitea/gitea/pull/3348/commits/047e8a258193f914ba9b66bd9bd1d80a127aefd3